app/src/main/java/fr/free/nrw/commons/nearby/NearbyListFragment.java

```diff
@@ -1,5 +1,6 @@
 package fr.free.nrw.commons.nearby;
 
+import android.support.v4.app.FragmentManager;
```

Comment:
Why did you import this?

Reply:
I was trying different solution at first. Will remove this line.